public/docs/ts/latest/guide/architecture.jade

```diff
@@ -238,20 +226,20 @@ figure
 <p style="padding-top:10px">Metadata tells Angular how to process a class.</p>
 <br clear="all">
 :marked
-  [Looking back at the code](#component-code) for `HeroListComponent`, we see that it's just a class.
+  [Looking back at the code](#component-code) for `HeroListComponent`, you see that it's just a class.
   There is no evidence of a framework, no "Angular" in it at all.
 
-  In fact, it really is *just a class*. It's not a component until we *tell Angular about it*.
+  In fact, it really is *just a class*. It's not a component until you *tell Angular about it*.
 
   We tell Angular that `HeroListComponent` is a component by attaching **metadata** to the class.
 
-  In !{_Lang}, we attach metadata by using !{_a} **!{_decorator}**.
+  In !{_Lang}, you attach metadata by using !{_a} **!{_decorator}**.
   Here's some metadata for `HeroListComponent`:
 
 +makeExcerpt('app/hero-list.component.ts', 'metadata')
 
 :marked
-  Here we see the `@Component` !{_decorator} which (no surprise) identifies the class
+  Here you see the `@Component` !{_decorator} which (no surprise) identifies the class
```

Comment:
I'd delete "(no surprise)" and firm up the writing a bit. How about:

Here is the ...}, which identifies...